modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -195,6 +200,54 @@ interfaces:
       auto-dns: true
 # ...
 ----
+<1> If you are using this configuration with dnsmasq or another DNS caching system, consider using `127.0.0.1`.
+
+* Configure a status DNS search option using Dynamic Host Configuration Protocol (DHCP) and IPv6 Stateless Address AutoConfiguration (SLAAC) servers.
```

Comment:
should be 'Configure a static DNS search option using ...'

Reply:
I am wondering whether it is supported or not in OpenShift for using `dnsmasq` on `127.0.0.1`.

Reply:
Removed this option and only included .Static DNS for network interface with automatic IP configuration